src/chat/Chat.js

```diff
@@ -46,3 +51,9 @@ export default class Chat extends React.Component {
     );
   }
 }
+
+const mapStateToProps = (state) => ({
+  readIds: state.flags.read
```

Comment:
Move this to MainScreenContainer. We have all the state retrieval there and we pass it down.
We may think about moving them around a bit, but for now move there?